src/controller/UserController.php

```diff
@@ -33,72 +32,32 @@ public function __construct(TwigService $twig)
         // Je stock la configuration du service twig dans notre variable twig du controller
         $this->twigService = $twig;
     }
-    
-    public function showUsers()
-    {
-        $userModel = new UserModel();
-        $users = $userModel->getUsers();
-        // On récupère des faux utilisateurs, une fois le UserModel fonctionnel
-        // Tu peux remplacer la ligne ci-dessous par les lignes 51 et 52.
-        // La fonction mockUsers() est just pour tester sans la base de données.
-        //$users = $this->mockUsers();
 
-        // On affiche les utilisateurs dans le template twig user.html.twig
-        echo $this->twigService->get()->render('user.html.twig', ['users' => $users]);
-    }
+    // public function showUsers()
+    // {
+    //     $userModel = new UserModel();
+    //     $users = $userModel->getUsers();
+    //     echo $this->twigService->get()->render('user.html.twig', ['users' => $users]);
+    // }
 
+    /**
+     * @return [type]
+     */
     public function showPosts(){
         $userModel = new UserModel();
         $posts = $userModel->getPosts();
-        // il faudra que je pointe vers le menu portfolio
         echo $this->twigService->get()->render('posts.html.twig', ['posts' => $posts]); 
     }
 
+    /**
+     * @param mixed $id
+     * 
+     * @return [type]
+     */
     public function showPostAndComments($id){
```

Comment:
`$id `ne peut pas être mixed. En POO chaque variable doit être typer. `int`, `string` ? 
```suggestion
    public function showPostAndComments(string $id) {
```